lib/cli.js

```diff
@@ -124,9 +125,14 @@ exports.run = () => {
 	if (cli.flags.tap && !cli.flags.watch) {
 		reporter = tapReporter();
 	} else if (cli.flags.verbose || isCi) {
-		reporter = verboseReporter();
+		reporter = verboseReporter({
+			disableColors: cli.flags.color !== undefined
```

Comment:
I will double check this. Was stuck on this point for a while trying out different things and reading up on meow and minimist. Got it working on undefined, but maybe I missed something.